recipes/wasmtime/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+import os
+import shutil
+
+required_conan_version = ">=1.33.0"
+
+class WasmtimeConan(ConanFile):
+    name = 'wasmtime'
+    homepage = 'https://github.com/bytecodealliance/wasmtime'
+    license = 'Apache-2.0'
+    url = 'https://github.com/conan-io/conan-center-index'
+    description = "Standalone JIT-style runtime for WebAssembly, using Cranelift"
+    topics = ("webassembly", "wasm", "wasi")
+    settings = "os", "compiler", "arch"
+    options = { "shared": [True, False] }
+    default_options = { 'shared': False }
+    no_copy_source = True
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "apple-clang": "9.4",
+            "clang": "3.3",
+            "gcc": "5.1"
+        }
+
+    @property
+    def _sources_key(self):
+        if self.settings.compiler == "Visual Studio":
+            return "Windows"
+        elif self.settings.os == "Windows" and self.settings.compiler == "gcc":
+            return "MinGW"
+        return str(self.settings.os)
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
```

Comment:
Looks like the hooks don't complain anymore about fPIC.
Maybe it's safe to remove their handling in `config_options` and `configure`.

Reply:
Removed